liquibase-standard/src/main/java/liquibase/util/StringUtil.java

```diff
@@ -1171,4 +1172,13 @@ public static List<String> splitToChunks(String input, int chunkSize) {
         }
         return chunks;
     }
+
+    /**
+     *
+     * Remove the "file://" prefix from a URI if it exists.
+     *
+     */
+    public static String stripUriPrefix(URI uri) {
+        return uri.toString().replace("file://","");
```

Comment:
I've updated this to check for null and return an empty string. I've also migrated this out of `StringUtil` since the only usage was in a single `InitProject` class. It can stay private for now.